packages/@aws-cdk/aws-eks/lib/ami.ts

```diff
@@ -21,6 +23,8 @@ export interface EksOptimizedAmiProps {
 
 /**
  * Source for EKS optimized AMIs
+ *
+ * @deprecated see {@link EksOptimizedImage}
```

Comment:
Since this is an experimental module, let’s just remove this API and the Ami.ts file.

Reply:
Done